src/Context/CountriesDataContext.tsx

```diff
@@ -34,9 +51,15 @@ export const CountriesDataContext = ({ children }: { children: ReactNode }) => {
 		return { error, countries };
 	}, [error, countries]);
 
+	const selectedCountryContext = useMemo(() => {
+		return { selectCountryRegion, setSelectCountryRegion };
+	}, [selectCountryRegion, setSelectCountryRegion]);
+
 	return (
 		<CountriesContext.Provider value={contextValues}>
-			{children}
+			<SelectCountriesContext.Provider value={selectedCountryContext}>
```

Comment:
One question - what do You think about moving this logic to already existing context without creating a new one? 🤔  There are just 2 new things to be added. SelectedCountry string  and setSelectedCountry  function. That;s it. Maybe let's stick to one context?

Reply:
changed. and still working :D